File: Java/different.java
/*
 *	Problem: A Different Problem
 */

import java.util.Scanner;
import java.lang.Math;

public class different {
	
	/*-----main-----*/
	public static void main(String[] args) {		
		Scanner sc = new Scanner(System.in);
		
		while ( sc.hasNext() ) {
			long a = sc.nextLong();
			long b = sc.nextLong();
			
			long absDiff = Math.abs(a - b);
			System.out.println(absDiff);
		}

	} // end main
	
}

File: JavaScript/different.js
/*
 *	Problem: "A Different Problem"
 */
 
 const readline = require('readline');
 
 const rl = readline.createInterface({
	 input: process.stdin,
	 output: process.stdout
 });
 
 rl.on('line', (line) => {
	 var ab = line.split(' ');
	 var a = parseInt(ab[0]);
	 var b = parseInt(ab[1]);
	 
	 var output = Math.abs( a - b );
	 console.log(output);	 
 });